docs/docsite/rst/user_guide/index.rst

```diff
@@ -10,19 +10,17 @@ User Guide
 
     Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. We ask that you open an issue or pull request if you come upon a term that we have missed. For more details, see `our CTO Chris Wright's message <https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language>`_.
 
-Welcome to the Ansible User Guide! This guide covers how to work with Ansible, including using the command line, working with inventory, interacting with data, writing tasks, plays, and playbooks; executing playbooks, and reference materials. This page outlines the most common situations and questions that bring readers to this section. If you prefer a traditional table of contents, you can find one at the bottom of the page.
+Welcome to the Ansible User Guide! This guide covers how to work with Ansible, including using the command line, working with inventory, interacting with data, writing tasks, plays, and playbooks; executing playbooks, and reference materials.
+Quickly find answers in the following sections or expand the table of contents below to scroll through all resources.
 
-Getting started
-===============
+Getting started with Ansible
+============================
 
-* I'd like an overview of how Ansible works. Where can I find:
+Quickly learn the basics to get up and running with Ansible.
+You can then build an inventory before creating and running your first playbook.
 
-  * a :ref:`quick video overview <quickstart_guide>`
-  * a :ref:`text introduction <intro_getting_started>`
+:ref:`Start using Ansible in a few easy steps <intro_getting_started>`.
```

Comment:
Initially I made a bunch of other changes but reverted them because I didn't want to cram too much in. What about removing that quickstart video? Can you even get that to play? I tried with different browsers but never could actually view it.